### ISSUE TITLE
Sqlproxy hasCredentials logic

### ISSUE THREAD

**Issue by EamonKeane** [rimusz/charts#45]
For sqlproxy to default to no `credentials.json`, `usingGCPController: "false"` should instead be set to `usingGCPController: ""` in `values.yaml`.

```
{{- define "gcloud-sqlproxy.hasCredentials" -}}
{{ or .Values.serviceAccountKey ( or .Values.existingSecret .Values.usingGCPController ) -}}
{{- end -}}
```

```
        {{ if $hasCredentials -}}
        - -credential_file=/secrets/cloudsql/{{ include "gcloud-sqlproxy.secretKey" . }}
        {{ end -}}
```

https://github.com/rimusz/charts/blob/master/stable/gcloud-sqlproxy/templates/_helpers.tpl#L58-L60

https://github.com/rimusz/charts/blob/master/stable/gcloud-sqlproxy/templates/deployment.yaml#L35-L37

**Comment by ianks** [comment]
yup, this is breaking my attempt to upgrade to this chart

**Comment by rimusz** [comment]
it was fixed in chart v0.13.1
